### DISCUSSION TITLE
How to call view function with ABI? Web3.js EVM Polygon

### DISCUSSION THREAD

**Discussion by thequantumdomain** [Cyfrin/foundry-full-course-cu#1533] [Q&A]
Issue unresolved and removed

**Comment by EngrPips** [comment]
Hello @thequantumdomain , have you checked out web3.js  documentation maybe there are stuffs that explained what you need in there.

**Reply by thequantumdomain** [reply]
@EngrPips  Hi. Sure. I am unable to find the specific instructions in the documentation. 
Do you have the expertise to analyse my code and suggest improvements?
Thanks

**Reply by EngrPips** [reply]
Been very long since i work with web3.js library , i can hardly remember what it is used for. Arrgh. Have you tried browsing google ?
